docusaurus/docs/reference/config-types/host.v2.md

```diff
@@ -0,0 +1,291 @@
+      succeed
+
+**Examples**
+
+```json
+{
+  "terminators": [
+    {
+      "forwardAddress": true,
+      "allowedAddresses": [
+        "192.168.1.0/24",
+        "10.0.0.1/16"
+      ],
+      "forwardPort": true,
+      "allowedPorts": [
+        {
+          "low": "1024",
+          "high": 2048
+        }
+      ],
+      "forwardProtocol": true,
+      "allowedProtocols": [
+        "tcp",
+        "udp"
+      ]
+    }
+  ]
+}
+```
+
+Note that not everything must be forwarded. For example the address could be hard coded, while the
+port and protocol could be forwarded.
+
+```json
+{
+  "terminators": [
+    {
+      "address": "192.168.100.1",
+      "forwardPort": true,
+      "allowedPorts": [
+        {
+          "low": "1024",
+          "high": 2048
+        }
+      ],
+      "forwardProtocol": true,
+      "allowedProtocols": [
+        "tcp",
+        "udp"
+      ]
+    }
+  ]
+}
+```
+
+Health checks and listen options also can be specified for each terminator.
+
+* `listenOptions` - Provides ways to customize the terminator
+    * See below for the full definition
+* `portChecks` - TCP port health check definitions
+    * See below for the full definition
+* `httpChecks` - HTTP health check definitions
+    * See below for the full definition
+
+## Listen Options
+
+* `bindUsingEdgeIdentity` - Associate the hosting terminator with the name of the hosting tunneler's
+  identity. Setting this to 'true' is equivalent to setting 'identity=$tunneler_id.name'",
+    * Boolean value, defaults to `false`
+* `connectTimeout` - Timeout when making connections to the external server. Specified as a
+  duration. Examples: `500ms`, `5s`, `1m`
+* `cost` - Static cost of the terminator. Must be a value between 0 and 65535. Default to 0.
+* `identity` - Associate the hosting terminator with the specified identity.
+    * '$tunneler_id.name' resolves to the name of the hosting tunneler's identity.
+    * '$tunneler_id.tag[tagName]' resolves to the value of the 'tagName' tag on the hosting
+      tunneler's identity.
+* `maxConnections` - Number of routers to create terminators on. Does not apply to hosting edge
+  routers, only to SDK hosted tunnelers.
+* `precedence` - Initial terminator precedence. Must be one of `default`, `required` or `failed`.
+  Defaults to `default`.
+
+**Example**
+
+```json
+{
+  "terminators": [
+    {
+      "address": "192.168.100.1",
+      "port": 80,
+      "protocol": "tcp",
+      "listenOptions": {
+        "connectTimeout": "1s",
+        "cost": 200,
+        "precedence": "required"
+      }
+    }
+  ]
+}
+```
+
+## Health Checks
+
+There are two kinds of health checks supported, port check and http checks.
+
+### Port Checks
+
+Port checks verify if a given port is accepting TCP connections. The check does not attempt to send
+or receive any data. Port check definitions support the following properties:
+
+* `address` - an IP or DNS address with port.
```

Comment:
you capitalized IP here but didn't a couple of times above.  Probably best to use IP and IPs?

Reply:
fixed